1.8/development/index.md

```diff
@@ -1,16 +1,103 @@
+
+# DC/OS Service structure
+
+Each DC/OS service contains `package.json`, `config.json`, and `marathon.json` files. The contents of these files are described in the DC/OS Service specification.
+
+<!-- This information should be replaced with link to service spec. JSH 11/23/15 -->
+
+*   **package.json**
+    
+    *   The `"name": "cassandra",` parameter specified here defines the DC/OS service name in the package repository. The must be the first parameter in the file. 
+    *   Focus the description on your service. Assume that all users are familiar with DC/OS and Mesos.
+    *   The `tags` parameter is used for user searches (`dcos package search <criteria>`). Add tags that distinguish your service in some way. Avoid the following terms: Mesos, Mesosphere, DC/OS, and datacenter. For example, the unicorns service could have:
+        
+              "tags": ["rainbows", "mythical"]
+            
+    
+    *   The `preInstallNotes` parameter gives the user information they'll need before starting the installation process. For example, you could explain what the resource requirements are for your service.
+        
+              "preInstallNotes":"Unicorns take 7 nodes with 1 core each and 1TB of ram."
+            
+    
+    *   The `postInstallNotes` parameter gives the user information they'll need after the installation. Focus on providing a documentation URL, a tutorial, or both. For example:
+        
+              "postInstallNotes": "Thank you for installing the Unicorn service.nntDocumentation: http://<your-url>ntIssues: https://github.com/",
+            
+    
+    *   The `postUninstallNotes` parameter gives the user information they'll need after an uninstall. For example, further cleanup before reinstalling again and a link to the details. A common issue is cleaning up ZooKeeper entries. For example:
+        
+              postUninstallNotes": "The Unicorn DC/OS Service has been uninstalled and will no longer run.nPlease follow the instructions at http://<your-URL> to clean up any persisted state" }
+            
+
+*   **config.json**
+    
+    *   The requirement block is for all properties that are required by the marathon.json file without a condition block (it is NOT properties that are not provided and thus must be supplied by the user)
+
+*   **marathon.json**
+    
+    *   A second-level (nested) property must be the framework-name with a value of the service name. For example:
+        
+              "framework-name" : "{{unicorn-framework-name}}"
+            
+    
+    *   Use the same value for the id parameter. For example:
+        
+              "id" : "{{unicorn-framework-name}}"
+            
+    
+    *   All URLs used by the service must be passed to the service by using command line or environment variable
```

Comment:
i think this entire section should be removed and we should point to https://github.com/mesosphere/universe#creating-a-package. thoughts?